0-bootstrap/main.tf

```diff
@@ -140,21 +133,6 @@ module "cloudbuild_bootstrap" {
     "billingbudgets.googleapis.com"
   ]
 
-  project_labels = {
-    environment       = "bootstrap"
-    application_name  = "cloudbuild-bootstrap"
-    billing_code      = "1234"
-    primary_contact   = "example1"
-    secondary_contact = "example2"
-    business_code     = "abcd"
-    env_code          = "b"
-  }
-
-  terraform_apply_branches = [
```

Comment:
The branch names cannot be changes without changing the wrapper script

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1d5b05bf35e98d62b4145a349d490df872449dee/build/tf-wrapper.sh#L26

Reply:
@daniel-cit 
should I add all above constraints into corresponding README files? 
As soon as we will move that data to variables forking and future code updates become easier I suppose.